toolkit/scripts/containerized-build/resources/mariner.Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG container_img
+FROM ${container_img}
+ARG version
+ARG enable_local_repo
+ARG topdir
+ARG mariner_repo
+ARG mariner_branch
+ARG splash_txt
+LABEL containerized-rpmbuild=$mariner_repo/build
+
+COPY resources/_local_repo /etc/yum.repos.d/local_repo.not_a_repo
```

Comment:
Perhaps `disabled_repo` instead of `not_a_repo`?

Reply:
That or just place the file in `mariner_setup_dir` to begin with, with its proper name, and then move it into `yum.repos.d` when appropriate.